### DISCUSSION TITLE
Environment pass before environment assignment

### DISCUSSION THREAD

**Discussion by hombit** [pypa/cibuildwheel#1609] [Q&A]
It looks like currently environment is set by `CIBW_ENVIRONMENT` before it is being updated from `CIBW_ENVIRONMENT_PASS_LINUX`. Could I make it work in the reverse order? I'm trying to do something like that:
```
CIBW_ENVIRONMENT="A=$B"
CIBW_ENVIRONMENT_PASS_LINUX=B
```

**Comment by joerick** [comment]
Yeah, I agree, that looks like an oversight. I'll make a PR.
